Automattic/jetpack
# Move debug helper to packages folder

After exploring a new action to maintain the mirror repo for the debug helper plugin up to date, we decided it was better to treat it as a package and let the current GH action that handles packages to handle this plugin as well.

This pseud-package will not be added to composer.json and will be mapped as a docker volume to be used in our local docker environment.

This PR

* Moves the plugin from `docker/mu-plugins` to `packages`
* adds a new volume to our `docker-compose.json`
* changes the GH action script to only check for `composer.json` file in the package if that file existed before.

#### Testing instructions:






Not possible to test since it will only work on master. We can run a test once this PR is merged.

I've tested it on my fork.

This commit: https://github.com/leogermani/jetpack/commit/6ea348d5048d4437ea1a62c744dd15f8208febc0

Triggered this action: https://github.com/leogermani/jetpack/runs/832123597?check_suite_focus=true

which resulted in an error, because the bot didn't have permission to push, but we can see from the log that the changes to the script worked as expected:
* current packages were treated as expected
* the check for `composer.json` worked for all repos
